src/components/AlertComponent.jsx

```diff
@@ -7,7 +7,7 @@ const AlertComponent = ({ variant, heading, text }) => {
   return (
     isShow && (
       <Alert
-        className='mb-0 px-2 px-md-3'
+        className='mb-0'
```

Comment:
Revert these changes! Because text is too long.
Do this!

`className='mb-0 px-2 px-md-3'`